pkg/util/azureclient/mgmt/insights/insights/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change History
+
+* Code from github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights
+* Modified target API version due to known fault in API responses, see https://github.com/Azure/azure-sdk-for-go/issues/14704 for more details
+* Revert when responses for ActivityLogs contain operationName: "Microsoft.Compute/virtualMachines/redeploy/action" again
```

Comment:
I would also add a reminder to remove this package so we do not have a dead code hanging around.

Reply:
`Revert` would remove the package.